src/App.js

```diff
@@ -19,34 +17,31 @@ const App = () => {
   return (
     <Router>
       <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1>TCL-16</h1>
-          <p>
-            Welcome to TCL-16! This app component is a good starting place for
-            the Smart Shopping List!
-          </p>
-          <h2>Week 1</h2>
-          <MenuLink activeWhenExact={true} pathTo="/" label="List View" />
-          <MenuLink pathTo="/AddItem" label="Add Item" />
+        {localStorage.getItem('token') ? (
```

Comment:
This is hardly authentication, but in place of it, we can switch on the presence of the user being 'authenticated' here a bit higher in the component hierarchy so we don't need to worry about controlling for the user's authentication state it in so many components.